src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -396,14 +440,17 @@ public Rotation2d getRotation() {
 
   /** Resets the current odometry pose. */
   public void setPose(Pose2d pose) {
-    poseEstimator.resetPosition(rawGyroRotation, getModulePositions(), pose);
+    poseEstimator.resetPosition(
+        rawGyroRotation.plus(new Rotation2d(Math.PI)), getModulePositions(), pose);
   }
 
   /** Adds a new timestamped vision measurement. */
   public void addVisionMeasurement(
       Pose2d visionRobotPoseMeters,
       double timestampSeconds,
       Matrix<N3, N1> visionMeasurementStdDevs) {
+
+    System.out.println(visionRobotPoseMeters);
```

Comment:
Same thing -- if we need these debugging helpers then so be it, but stable code should probably purge them once tested.